### PR TITLE
Set up project

### PR DESCRIPTION
**Pull request by seddonym**
Set up a few basics for the project - dev requirements, docs, license, github actions etc. 

# What this builds on

Builds on [this commit](https://github.com/kraken-tech/python-rustfluent/commit/c9ba4683d72a131d9ad558f7f97a7939a543d360), which is at the time of writing the only commit on the project.

That commit was generating using using `cookiecutter gh:octoenergy/cookiecutter-kraken-package`.

I changed one small thing before committing it: I renamed the package from `kraken.rustfluent` just to `rustfluent` as it didn't feel the namespace was appropriate in this case.

### CODE REVIEW

File: noxfile.py
Comment (seddonym) [inline noxfile.py:1]: This is just following the instructions in the cookie cutter template.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
+# Contributing
```
Comment (seddonym) [inline CONTRIBUTING.md:1]: All of these docs are not relevant to consumers of the package, only to contributors.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1 @@
+* @kraken-tech/open-source-maintainers
```
Comment (seddonym) [inline .github/CODEOWNERS:1]: Not sure why this is invalid, it's fine in [this repo](https://github.com/kraken-tech/timezone-tools/blob/main/.github/CODEOWNERS).

File: README.md [hunk 3]
```diff
@@ -1,199 +1,52 @@
 # rustfluent
 
-TODO describe the purpose of 'rustfluent' here.
+A Python interface to the Rust Fluent Library.
+
+This project is a shall shim around [fluent-rs](https://github.com/projectfluent/fluent-rs), so it
```
Comment (khorn) [inline README.md:5]: Typo: 'shall' -> "small"

File: .github/CODEOWNERS (hunk 2 above)
Comment (khorn) [reply .github/CODEOWNERS:1]: It shows valid for me?

**Review by dooferlad — APPROVED** [review]

**Review by khorn — APPROVED** [review]
Tiny typo, otherwise :+1:

File: .github/CODEOWNERS (hunk 2 above)
Comment (seddonym) [reply .github/CODEOWNERS:1]: Oh great, it is now. I realised there were some github set up steps I needed to complete, as documented in the cookiecutter readme. https://github.com/octoenergy/cookiecutter-kraken-package?tab=readme-ov-file#configure-github

**Pull request merged**